### PR TITLE
[Android] FirstVoices app

### PR DESCRIPTION
**Pull request by mcdurdin**
This currently builds and runs. However, keyboards are missing and various other items need tidyup.

### CODE REVIEW

**Comment by keyman-server** [comment]
[OSK renders for Windows](https://build.palaso.org/repository/download/KeymanDesktop_TestPrRenderOnScreenKeyboards/152373:id/kvk/index.html)

File: oem/firstvoices/android/build_keyboards.sh [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  echo "  -clean-keyboards: Clean the keyboards from this repo"
+  echo "  -debug: Build debug versions of the keyboards"
+  exit 1
+}
+
+DO_CLEAN=true
+DO_COPY=true
+DO_BUILD=true
+
+while [[ $# -gt 0 ]] ; do
```
Comment (darcywong00) [inline oem/firstvoices/android/build_keyboards.sh]: I tried building this in Ubuntu 

1. I had to make the 3 .sh files executable `sudo chmod +x *.sh`
2. Is this line bash syntax?
```
while [[ $# -gt 0 ]] ; do
```

Comment (darcywong00) [reply oem/firstvoices/android/build_keyboards.sh]: and I had to make `gradlew` executable

Comment (mcdurdin) [reply oem/firstvoices/android/build_keyboards.sh]: 👍  all done.

Point 2 above? I've updated the relevant scripts to use `#!/bin/bash`. Were you having trouble with it in other shells?

**Pull request merged**